scripts/lib/zulip_tools.py

```diff
@@ -580,6 +580,24 @@ def get_deploy_options(config_file: configparser.RawConfigParser) -> List[str]:
     return get_config(config_file, "deployment", "deploy_options", "").strip().split()
 
 
+def run_psql_as_postgres(
+    sql_query: str,
+) -> None:
+    subcmd = " ".join(
+        map(
+            shlex.quote,
+            [
+                "psql",
+                "-v",
+                "ON_ERROR_STOP=1",
```

Comment:
This dorps the `zulip`  database name that was in this position.  I think we should preserve that text here until upcoming refactoring changes it.

Reply:
I merged this commit as 21cc186105b3ad7d8988bb86fded3caa1f42a5d6 after making that small tweak, just to simplify this PR.

Reply:
I'll rebase this and push it again now.